.circleci/config.yml

```diff
@@ -2,7 +2,12 @@
 #
 # Check https://circleci.com/docs/2.0/language-ruby/ for more details
 #
-version: 2
+version: 2.1
+
+orbs:
+  jq: circleci/jq@2.2.0
```

Comment:
Necessary?

Reply:
No, this isn't, copy-pasta from IDP. I'll remove